evennia/contrib/utils/database_backup/README.md

```diff
@@ -0,0 +1,69 @@
+# Database Backup Scheduler
+
+Contribution by helpme (2024)
+
+This module schedules backups in-game, which saves a copy of your database to your game's `server/backups` folder. Database backups are *not* automatically uploaded to any cloud service, it is left to you to decide what to do with them (i.e. pushed to git, uploaded to the ether, downloaded to a hard drive).
+
+Backups can take place at any time. Restoring the game world from backup takes place during downtime, as documented below.
+
+Currently, the sqlite3 (the evennia default) and postgresql databases are supported. Others are welcome to add more.
+
+## Installation
+
+This utility adds the `backup` command. Import the module into your commands and add it to your command set to make it available.
```

Comment:
Since this is based on a global Script, why not also describe how to just add the script as a global script in settings; that way it could be set up without having to connect to the game and run a command ...